src/components/genres/Genres.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.css';
+
+const Genres = (props) => {
+    return (
+        <ul className={styles.genres}>
+            {props.genres.map((item) => (
+                <li key={item}>
```

Comment:
Even though you have an array of strings inside `genres` and feeling safe here, in general it's not recommended to assign direct value of an array to a `key` attribute. One of the common cases is using `index` of an array item mixed with some string value, e.g. 
``{props.genres.map((item, index) => (<li key=`genre-${index}`>)``
Or it's ok to assign unique value (id) mixed with string instead of index, as you have in `MoviesList.jsx`